src/views/search/controls/SearchControls.tsx

```diff
@@ -102,11 +103,14 @@ export const SearchControls = ({ hasQuery }: Props) => {
             <VuiFlexContainer alignItems="center" spacing="m">
               {searchHeader.description && (
                 <VuiFlexItem grow={false}>
-                  <VuiTitle size="xxs" align="right">
+                  <VuiTitle size="xs" align="right">
                     <VuiTextColor color="subdued">
-                      <h2 style={{ whiteSpace: "pre-line" }}>
-                        {searchHeader.description.replaceAll("\\n", "\n")}
-                      </h2>
+                      <Markdown
```

Comment:
Markdown is immensely powerful and will enable elements here that don't belong, such as lists, tables, and multiple paragraphs.

I think a more targeted solution to the stated problem would be to add a `searchHeader.descriptionUrl` option. Then this code would conditionally render a link if that option is defined. What do you think of this approach?

```tsx
const searchHeaderDescription = searchHeader.description.replaceAll("\\n", "\n");

const searchHeaderContent = searchHeader.descriptionUrl ? (
  <a href={searchHeader.descriptionUrl}>{searchHeaderDescription}</a>
) : searchHeaderDescription;

return (
  <VuiTextColor color="subdued">
    <h2 style={{ whiteSpace: "pre-line" }}>
      {searchHeaderContent}
    </h2>
  </VuiTextColor>
);
```

Reply:
So in this way the whole description is a single URL right?
In this case, the idea was to have something like "Created by Bob Gourley (OODA)" where only OODA is a URL linking to Bob's website. Any good way to do that?
I think Markdown can also be useful to do multi-line descriptions that we want to control the line breaks e.g. with <br>

Reply:
I think I'd probably make the whole thing a link or I'd add a location in the UI for rendering some kind of authorship link. I'll approve this PR to unblock you though.